src/Illuminate/Support/functions.php

```diff
@@ -13,7 +13,7 @@
      * @param  callable|null  $callback
      * @param  string|null  $name
      * @param  bool  $always
-     * @return \Illuminate\Support\Defer\DeferredCallback
+     * @return \Illuminate\Support\Defer\DeferredCallback|\Illuminate\Support\Defer\DeferredCallbackCollection
```

Comment:
[You can narrow this even more down if you want](https://phpstan.org/blog/phpstan-1-6-0-with-conditional-return-types):
```suggestion
     * @return ($callback is null ? \Illuminate\Support\Defer\DeferredCallbackCollection : \Illuminate\Support\Defer\DeferredCallback)
```